### ISSUE TITLE
Fix mDNS on ReFactor

### ISSUE THREAD

**Issue by goeland86** [goeland86/ReFactor#17]
The mDNS resolution doesn't work on ReFactor yet.
Make sure that avahi is installed and properly setup for the image, and adjust the OctoPrint config to use the webcam feed through mDNS resolution.

**Comment by electronlove** [comment]
The issue I was having with the Octoprint config to use the webcam feed is that /usr/local/bin/mjpg_streamer is not there and required a manual install from /usr/src/mjpg-streamer/mjpg-streamer-experimental + changing the arm.local to beaglebone.local.

**Comment by goeland86** [comment]
@electronlove Understood - but is mDNS working for OctoPrint itself though?

**Comment by electronlove** [comment]
Yes.  I can open a webpage and navigate to "http://beaglebone.local:5000" and it pulls up Octoprint - the Discovery plugin seems to be working and avahi is running with no errrors.   On a side, read that there was also a potential mDNS issue with Windows 10 which could be fixed by installing ITunes...odd indeed but I did that.  Also note that for redeem RC3, was able to pull up OctoPrint from http://kamikaze.local:5000 as well and the webcam shows up properly without having to feed an ip address to it.
